Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -674,7 +674,35 @@ def fetch_issues(max_fetch):
         {'time': datetime.now().strftime(DEMISTO_OCCURRED_FORMAT)})
 
 
-def get_filtered_issues(issue_type, resource_id, severity, limit, issue_id=''):
+def get_issue(issue_id):
+    demisto.info(f"Issue id is {issue_id}\n")
+
+    is_valid_id, message = is_valid_issue_id(issue_id)
+    if not is_valid_id:
+        demisto.error(message)
+        return message
+
+    issue_variables = {
+        "first": 5,
+        "filterBy": {
+            "id": issue_id
+        }
+    }
+
+    response_json = checkAPIerrors(PULL_ISSUES_QUERY, issue_variables)
+
+    demisto.info(f"The API response is {response_json}")
+
+    issues = {}
+    if response_json['data']['issues']['nodes'] != []:
+        issues = response_json['data']['issues']['nodes']
+    else:
+        demisto.error(f"There was no result for Issue ID: {issue_id}")
+
```

Comment:
```suggestion
    issues = response_json.get('data', []).get('issues', []).get('nodes', {})
```

Reply:
I also get this warning from my code editor, but the code doesn't work the same way when refactored this way. As we only implemented a minor change for this version, I prefer not modifying these lines right now.